components/Pagination/src/pagination.css

```diff
@@ -0,0 +1,114 @@
+.denhaag-pagination {
+  --denhaag-pagination-align: center;
+
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: var(--denhaag-pagination-align);
+}
+
+@media (min-width: 640px) {
+  .denhaag-pagination,
+  .denhaag-pagination--start {
+    --denhaag-pagination-align: flex-start;
+  }
+
+  .denhaag-pagination--center {
+    --denhaag-pagination-align: center;
+  }
+
+  .denhaag-pagination--end {
+    --denhaag-pagination-align: end;
+  }
```

Comment:
Why do all these variations exist? Can projects in The Hague choose freely, or are there recommendations around this? My general suggestion would be to offer the smallest API possible, because you'll need to support it.

Reply:
Hi @Robbert , since it wasn't defined in the Design, I added an option as modifier for all three alignments since It wasn't clear for me. From work experience I know that person A wants it on the left and Person B centert. 

Support for only a single `justify-content` shouldn't be that much (in my opinion).

Anyway I just submitted a request to design about how this should be aligned ( https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn?node-id=1673:231#131890306 )